tests/jetty-ee10-test-maven-plugin/src/it/jetty-cdi-start-forked/pom.xml

```diff
@@ -69,6 +69,13 @@
       <plugin>
         <groupId>org.eclipse.jetty.ee10</groupId>
         <artifactId>jetty-ee10-maven-plugin</artifactId>
+        <dependencies>
+          <dependency>
+              <groupId>org.eclipse.jetty</groupId>
+              <artifactId>jetty-slf4j-impl</artifactId>
+              <version>${jetty.version}</version>
```

Comment:
Don't we have this as `@jetty.version@`?
Should we have this in `it-parent-pom` like the other deps we use in these IT tests?